vulture/core.py

```diff
@@ -121,9 +125,15 @@ def __hash__(self):
 
 class Vulture(ast.NodeVisitor):
     """Find dead code."""
-    def __init__(self, exclude=None, verbose=False, sort_by_size=False):
+    def __init__(self, exclude=None, verbose=False, sort_by_size=False,
+                 min_confidence=0):
         self.exclude = []
         self.sort_by_size = sort_by_size
+        self.min_confidence = float(min_confidence or 0)
```

Comment:
I think I sound confusing. By default, I meant: `float(min_confidence or DEFAULT_CONFIDENCE)`
and by default arg, I meant:
```python
def __init__(self, ..., confidence=DEFAULT_CONFIDENCE # This won't work.
```